secretmanager/snippets/disable_secret_with_delayed_destroy.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python
+
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+
+import argparse
+
+from google.cloud import secretmanager
+
+
+# [START secretmanager_disable_secret_with_delayed_destroy]
+def disable_secret_with_delayed_destroy(
+    project_id: str, secret_id: str
+) -> secretmanager.Secret:
+    """
+    Disable the version destroy ttl on the given secret version.
+    """
+
+    # Import the Secret Manager client library.
+    from google.cloud import secretmanager
```

Comment:
fix: duplicate import, I suggest leaving the one on line 18 with the comment from line 29 and removal of lines 29 and 30.